pkg/tsdbctl/check.go

```diff
@@ -91,58 +126,172 @@ func (cc *checkCommandeer) check() error {
 	}
 
 	// Get metric data and metadata
-	allAttrs := append(cc.attrs, "__name", "_name", "_lset", "_maxtime")
 	container, tablePath := cc.rootCommandeer.adapter.GetContainer()
-	objPath := fmt.Sprintf("%s/%s", tablePath, cc.objPath)
+
+	var respChans []chan *v3io.Response
+	if cc.metricName == "" {
+		respChan, err := cc.checkByPath(container, tablePath)
+		if err != nil {
+			return errors.Wrapf(err, "failed to check using object path '%s'", cc.objPath)
+		}
+		respChans = []chan *v3io.Response{respChan}
+		objPath := "/" + path.Join("", tablePath, cc.objPath)
```

Comment:
change to `objPath := path.Join("/", tablePath, cc.objPath)` otherwise you may get "//" at the beginning.
Note `path.Join` already takes care of separators